File: 01Python/01_introduction/Task1/vowel.py
import sys

#declare vowels
Vowels = ('a','e','i','o','u')

#input from user
GivenLetter = input("Please enter your letter (e.g., h):")

#Check the input
if not GivenLetter.isalpha():
    print("ERROR: Your input is not a valid string format")
    sys.exit()

if not len(GivenLetter) == 1:
    print("ERROR: Your input is not a single letter")
    sys.exit()

GivenLetter = GivenLetter.lower()
VowelFlag = False
for iterator in Vowels :
    if iterator == GivenLetter :
        VowelFlag = True
        break

if VowelFlag == False : 
    print("Your input is not a vowel")
else :
    print("Your input is a vowel")

File: 02_C++/01_Introduction/Vowel.cpp
#include <iostream>
#include <string>

int main()
{
    const std::string vowels = "aeiou";
    char letter;
    std::cout<<"Enter charater : ";
    std::cin>>letter;
    
    if(vowels.find(letter)<vowels.length())
    {
        std::cout<<"It is a vowel";
    }
    else
    {
        std::cout<<"It is Not a vowel";
    }
    
    return 0;
}
